source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRShowCardVisibilityHandler.h

```diff
@@ -0,0 +1,22 @@
+//
+//  ACRShowCardVisibilityHandler.h
```

Comment:
Why create a new file for this? can you add the delegate to ACRActionDelegate as @optional?

Reply:
Because its a delegate for ACRShowCardTarget.mm, and you can only set this delegate if you are overriding showcard action renderer.
If I add it in ACRActionDelegate than any class which want to implement the delegate will have to Implement the required methods of ACRActionDelegate too.